### PR TITLE
Added some images in the GitHub Actions lab, refined instructions elsewhere

### PR DESCRIPTION
**Pull request by noahShinabarger**

### CODE REVIEW

File: gh-actions.md [hunk 1]
```diff
@@ -156,12 +157,7 @@ docker push ghcr.io/<Your GitHub Username>/mayhem-cmake-example:latest
     ![repo_selection](assets/images/add-repo.png)
     
     Lastly we need to set the permissions to "Write" instead of just "Read". In the "Manage Actions access" section, to the write of `mayhem-cmake-example` choose the role to be "Write".
-    ![write_selection](assets/images/set_to_write.png)
-
-6. Mark the package as public following these steps in GitHub: https://docs.github.com/en/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility#configuring-access-to-container-images-for-your-personal-account
```
Comment (xansec) [inline gh-actions.md:161]: Any reason we're omitting this chunk here?

Comment (Jon-FAS) [reply gh-actions.md:161]: The instructions are outdated and don't match the current GUI layout.

File: gh-actions.md [hunk 2]
```diff
@@ -156,12 +157,7 @@ docker push ghcr.io/<Your GitHub Username>/mayhem-cmake-example:latest
     ![repo_selection](assets/images/add-repo.png)
     
     Lastly we need to set the permissions to "Write" instead of just "Read". In the "Manage Actions access" section, to the write of `mayhem-cmake-example` choose the role to be "Write".
```
Comment (Jon-FAS) [inline gh-actions.md:159]: typo, should be "right" of ...

File: gh-actions.md (hunk 1 above)
Comment (Jon-FAS) [reply gh-actions.md:161]: The instructions were confusing to the students as they didn't match the GUI anymore.

**Review by Jon-FAS — CHANGES_REQUESTED** [review]
Some typos

**Comment by xansec** [comment]
LGTM!

**Pull request merged**